client/js/components/GameView/MoveChoiceOverlay.vue

```diff
@@ -155,6 +166,8 @@ export default {
                 if (!validTargetExists) {
                   oneOffDisabled = true;
                   oneOffDisabledExplanation = 'There are no Royals to target';
+                } else if (allMovesAreDisabled) {
+                  oneOffDisabledExplanation = allMovesDisabledExplanation;
```

Comment:
Interesting UX question. If all moves are disabled and also this specific move is disabled (like it's not your turn but even if it were your turn there's no valid target for your one-off), what message should we show?

I had imagined we would show the global one e.g. "It's not your turn", but this would prioritize the specific one "there's no valid target" instead. What do you think would be preferable?

Reply:
If we opt to switch the priority, which I'm leaning towards, all you'll need to do is swap the if and else if statements

Reply:
I also suggest swapping them fwiw.

Reply:
Yeah I think "It's not your turn" should have the highest priority. When it's not a players turn, that's all that really matters imo.

Reply:
Sounds good. I think with this and the .only() fix we're good to go